### PR TITLE
RailsAdmin Address edit

### PR DESCRIPTION
**Pull request by TheFehr**

### CODE REVIEW

File: app/models/address.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class Address < ApplicationRecord
+  has_one :organization, dependent: :restrict_with_exception
```
Comment (lukasbischof) [inline app/models/address.rb:4]: Wouldn't it more be like a nullify?

Comment (TheFehr) [reply app/models/address.rb:4]: i used what creditor detail used

Comment (TheFehr) [reply app/models/address.rb:4]: but i can change for sure

Comment (lukasbischof) [reply app/models/address.rb:4]: well I had to change creditor_detail in my admin pr

Comment (TheFehr) [reply app/models/address.rb:4]: so should i do :nullify?

Comment (TheFehr) [reply app/models/address.rb:4]: problem there is org currently has not null on address column

Comment (TheFehr) [reply app/models/address.rb:4]: so that would cause problems i'd say

Comment (lukasbischof) [reply app/models/address.rb:4]: ah ok

**Review by lukasbischof — APPROVED** [review]

**Pull request merged**